daprdocs/content/en/developing-applications/error-codes/http-error-codes.md

```diff
@@ -0,0 +1,21 @@
+---
+type: docs
+title: "Handling HTTP error codes"
+linkTitle: "HTTP"
+description: "Detailed reference of the Dapr HTTP error codes and how to handle them"
+weight: 30
+---
+
+For http calls made to Dapr runtime, when an error is encountered, an error json is returned in http response body. The json contains an error code and an descriptive error message, e.g.
+
+```
+{
+    "errorCode": "ERR_STATE_GET",
```

Comment:
Need to check, but I feel this error format is the same as the one above. List this. I need to check with Alice on this, or ask Jake or watch the video again

```json
{
  "error": "Bad Request",
  "error_msg": "{\"errorCode\":\"ERR_STATE_STORE_NOT_FOUND\",\"message\":\"state store <name> is not found\",\"details\":[{\"@type\":\"type.googleapis.com/google.rpc.ErrorInfo\",\"domain\":\"dapr.io\",\"metadata\":{\"appID\":\"nodeapp\"},\"reason\":\"DAPR_STATE_NOT_FOUND\"}]}",
  "status": 400

Reply:
@msfussell what I have, I grabbed from the video while it was paused